persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -42,6 +42,20 @@ CREATE INDEX message_status_index ON Message(status);
 CREATE INDEX message_expire_time_index ON Message(expire_after_millis);
 CREATE INDEX message_self_deletion_end_date ON Message(self_deletion_end_date);
 
+CREATE TABLE MessageLinkPreview (
```

Comment:
I think, we need to include this table in the migration of messages (backup).

`DumpContent.sq`
`ImportContent.sq`

Reply:
Also, this new table needs to have a Migration `sqm` file :D

Reply:
With your help and the help of https://cashapp.github.io/sqldelight/2.0.0-alpha05/multiplatform_sqlite/migrations/ i was able to add the parts for migration and backup. The `verifySqlDelightMigration` task is now successful.